src/Nethermind/Nethermind.Evm/CodeAnalysis/JumpDestinationAnalyzer.cs

```diff
@@ -15,26 +15,28 @@ public sealed class JumpDestinationAnalyzer
         private const int BEGINSUB = 0x5c;
         private const int BitShiftPerInt32 = 5;
 
-        private int[]? _codeBitmap;
+        private int[]? _jumpDestBitmap;
         public byte[] MachineCode { get; set; }
 
         public JumpDestinationAnalyzer(byte[] code)
         {
+            // Store the code refence as the JumpDest analysis is lazy
+            // and not performed until first jump.
```

Comment:
Question - should we start creating bitmap in the background task as soon as we get here? How much would it speed up things? How much not needed work would we do?

Reply:
Could assume if contract created would be called fairly soon after even if just constructor

Reply:
Added

Reply:
Yeah, I like that :)

Reply:
Yeah, I like that. First use of `IThreadPoolWorkItem` in the repo 😅

Reply:
I would prefer some stats gathering first :)